metabase/metabase
# Field Search in Bulk Filter Modal

## Description

Adds a search box that can be used to search for a field

![fieldSearch](https://user-images.githubusercontent.com/30528226/178071319-33b6cb07-1ba6-440c-a36c-dff3684c1546.gif)

## To Test

- Search for fields in primary table
- Search for fields in linked tables
- Search for aggregate fields
- Search for "segments" or a segment name
- Search for something invalid so that you see the empty state
- Apply filters from the search view and make sure they show up properly when the search is cleared or applied
- type really fast... or slow to make sure the search debouncing works and the UI stays responsive
